dvc/commands/queue/kill.py

```diff
@@ -11,20 +11,31 @@ class CmdQueueKill(CmdBase):
     """Kill exp task in queue."""
 
     def run(self):
-        self.repo.experiments.celery_queue.kill(revs=self.args.task)
+        self.repo.experiments.celery_queue.kill(
+            revs=self.args.task, force=self.args.force
+        )
 
         return 0
 
 
 def add_parser(queue_subparsers, parent_parser):
-    QUEUE_KILL_HELP = "Kill actively running experiment queue tasks."
+    QUEUE_KILL_HELP = "Send SIGINT(Ctrl-C) to running experiment queue tasks."
```

Comment:
Using signal names in these help strings might be a bit too detailed, not sure DVC users will understand the difference between sigint and sigterm/sigkill?

I'd maybe use something like `Gracefully interrupt running experiment queue tasks (equivalent to Ctrl-C)` for the general message

@dberenbaum can weigh in here